pkg/apis/configuration/validation/virtualserver_test.go

```diff
@@ -2394,6 +2394,46 @@ func TestValidateVirtualServerRouteSubroutesFails(t *testing.T) {
 			pathPrefix: "/abc",
 			msg:        "invalid prefix",
 		},
+		{
+			routes: []v1.Route{
+				{
+					Path: "/abc",
+					Action: &v1.Action{
+						Pass: "test-1",
+					},
+				},
+				{	Path: "~^/test",
+					Action: &v1.Action{
+						Pass: "test-1",
+					},
+				},
+			},
+			upstreamNames: map[string]sets.Empty{
+				"test-1": {},
+			},
+			pathPrefix: "/abc",
+			msg:        "invalid prefix",
+		},
+		{
+			routes: []v1.Route{
+				{
+					Path: "=/test",
+					Action: &v1.Action{
+						Pass: "test-1",
+					},
+				},
+				{	Path: "/abc",
+					Action: &v1.Action{
+						Pass: "test-1",
+					},
+				},
+			},
+			upstreamNames: map[string]sets.Empty{
+				"test-1": {},
+			},
+			pathPrefix: "/abc",
```

Comment:
I think `pathPrefix` might not accurately describe what we're testing anymore since we could be using prefix/exact/regex path for route in virtual server here.